src/libs/_tests/runtime-utils.test.js

```diff
@@ -1,9 +1,28 @@
 import { tools } from 'src/components/notebook-utils'
 import {
-  getAnalysesDisplayList, getCurrentApp, getCurrentAppDataDisk, getCurrentAppIncludingDeleting, getDiskAppType, workspaceHasMultipleApps, workspaceHasMultipleDisks
+  getAnalysesDisplayList, getCostDisplayForTool, getCurrentApp, getCurrentAppDataDisk, getCurrentAppIncludingDeleting, getDiskAppType, workspaceHasMultipleApps, workspaceHasMultipleDisks
 } from 'src/libs/runtime-utils'
 
 
+jest.mock('src/data/machines', () => {
```

Comment:
It's very cool to see how this mocking works!

Can you say more about why you decided to mock the return values for 'US-CENTRAL1'? I get that if these values change, the unit test would have to be updated, but I would expect to make that type of change.

Reply:
Nathan and I discussed this very question as he was getting started on this work.  My thinking was that over time (if not already), this config object being exported could be used in a number of areas and unit/functional tests.  It would be a bit onerous to have to update all of them for every little config change if the intrinsic shape/meaning of the object has no substantive impact on the consuming code's logic.  

So as long as the shape (contract) of the dependency isn't changing, the unit tests should not need updates.   It also becomes crisper to understand the minimal stub data required to drive the code being tested, and give the test file the ability to manufacture key variations of the dependency that may not be guaranteed to exist in the real object at a given time.  Also, this gets easier to manage with Typescript, which is why it's a topic I plan to raise for discussion at this week's UIWG.

 But the goal is to drive discussion and feedback :)